File: 프로그래머스/1/131705. 삼총사/삼총사.cs
using System;

public class Solution {
    public int solution(int[] number) {
        int answer = 0;
        for(int i = 0; i< number.Length-2 ;i++)
        {
            for(int j = i+1; j< number.Length-1;j++)
            {
                for(int r = j+1;r<number.Length;r++)
                {
                    if(number[i]+number[j]+number[r] == 0)
                        answer += 1; 
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int length = number.length;
        
        for(int left = 0; left < length - 2; left++)
        {
            for(int mid = left+1; mid < length - 1; mid++)
            {
                for(int right = mid+1; right < length; right++)
                {
                    if(number[left] + number[mid] + number[right] == 0)
                        answer++;
                }
            }
        }
        return answer;
    }
}